__tests__/virtual-pet.test.js

```diff
@@ -0,0 +1,217 @@
+// require Pet/BabyPet constructor functions
+const {
+    Pet,
+    BabyPet
+} = require('../src/virtual-pet.js');
+
+const ERROR_MESSAGE = 'Your pet is no longer alive :(';
+let fido, bronson;
+beforeEach(() => {
```

Comment:
Grouping all constants together just makes this a bit easier to read. Also the fido and bronson variables dont need to be declared here are you're doing it already in `beforeEach()`

```suggestion
const ERROR_MESSAGE = 'Your pet is no longer alive :(';

beforeEach(() => {
```